eth/utils.go

```diff
@@ -714,23 +703,26 @@ func ParseMultisigInput(tx *store.TransactionETH, networtkID int, multisigStore,
 		// TODO: send notfy to owners about -1 confirmation. store confirmations in db
 
 		tx.Multisig.Owners = []store.OwnerHistory{}
-
 		log.Debugf("revokeConfirmation: %v", tx.Multisig.Input)
-		i, _ := new(big.Int).SetString(tx.Multisig.Input, 16)
 
-		sel := bson.M{"multisig.requestid": i.Int64(), "multisig.contract": contract.ContractAddress}
+		requestid, err := parseRevokeInput(tx.Multisig.Input)
+		if err != nil {
+			log.Errorf("ParseMultisigInput:revokeConfirmation: parseRevokeInput %v requestid:%v  contract:%v ", err.Error(), requestid, contract.ContractAddress)
```

Comment:
would it be better to stop function propagation (i.e return) in case of error occured?